guides/source/error_reporting.md

```diff
@@ -119,35 +112,57 @@ rescue StandardError => e
 end
 ```
 
-#### Options
+Any options you pass will be passed on the error subscribers.
 
-All 3 reporting APIs (`#handle`, `#record`, and `#report`) support the same options:
+### Error-reporting options
 
-- `handled`: a `Boolean` to tell if the error was handled
-- `severity`: a `Symbol` about the severity of the exception. Expected values are: `:error`, `:warning`, and `:info`
-- `context`: a `Hash` to provide more context about the error, like request headers or record attributes
-- `source`: a `String` about the source of the exception. Default is `"application"`
-    - You can use it to skip exceptions from certain sources
+All 3 reporting APIs (`#handle`, `#record`, and `#report`) support the following options, which are then passed along to all registered subscribers:
 
-### Setting Context
+- `handled`: a `Boolean` to indicate if the error was handled. This is set to `true` by default. `#record` sets this to `false`.
+- `severity`: a `Symbol` describing the severity of the error. Expected values are: `:error`, `:warning`, and `:info`. `#handle` sets this to `:warning`, while `#record` sets it to `:error`.
+- `context`: a `Hash` to provide more context about the error, like request or user details
+- `source`: a `String` about the source of the error. The default source is `"application"`. Errors reported by internal libraries may set other sources; the Redis cache library may use `"redis_cache_store.active_support"`, for instance. Your subscriber can use the source to ignore errors you aren't interested in.
 
-In addition to setting context through the `context` option, you can also use the [`#set_context`](https://api.rubyonrails.org/classes/ActiveSupport/ErrorReporter.html#method-i-set_context) API. For example:
+```ruby
```

Comment:
nitpick: can we standardise on using `rb` instead of `ruby`?

Reply:
Right...my old habits